share/help.txt

```diff
@@ -8,18 +8,18 @@ Alt-T     Tune                          Alt-N   Note in log
 Alt-V     Band down (CW Speed?)         Alt-P   Turn PTT on/off
 Alt-H     Show help.txt                 Alt-W   CW Weight
 Alt-I     Show talk messages            Alt-J   See QRG's
-Alt-K     CW from Keyboard              Alt-X   Exit program (Alt-q)
-Alt-M     Multi display (Alt-c)         Alt-S   Score window (Alt-r)
-Alt-,     Show bandmap                  Alt-J   Show frequency
+Alt-K     CW from Keyboard              Alt-X   Exit program
+Alt-M     Multi display (Alt-C)         Alt-S   Score window (Alt-R)
 Alt-Z     Show worked zones             Alt-G   Grab call (min. 2 letters from bandmap)
+Alt-,     Show bandmap
```

Comment:
Why did you drop `Alt-J`. It still works and is useful in multi op operation.

Reply:
It's still there, just dropped the second one and moved Alt-, to the end of the list.
```
   Alt-J   See QRG's
   Alt-J   Show frequency
```

Reply:
Silly me. I see, it was there two times. You dropped only the second one.